resources/guidelines/testing/e2e-best-practises.md

```diff
@@ -77,3 +78,5 @@ cy.wait('@getData').then((xhr) => {
 });
 ```
 
+{% hint style="info" %} This [best practise](https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting) is also mentioned in Cypress' best practises as well. Actually, it can be considered as general best practise to avoid flakiness.
```

Comment:
```suggestion
{% hint style="info" %} This [best practice](https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting) is also mentioned in Cypress' best practices as well. Actually, it can be considered as general best practice to avoid flakiness.
```